geth/node/node_api_test.go

```diff
@@ -0,0 +1,54 @@
+package node
+
+import (
+	"testing"
+
+	whisper "github.com/ethereum/go-ethereum/whisper/whisperv6"
+	"github.com/status-im/status-go/geth/params"
+	"github.com/stretchr/testify/require"
+)
+
+func TestWhisperLightModeEnabledSetsEmptyBloomFilter(t *testing.T) {
+	config, err := params.NewNodeConfig("/tmp", params.StatusChainNetworkID, true)
+	require.NoError(t, err)
+	config.WhisperConfig.LightClient = true
+
+	// Node is ephemeral (only in memory).
```

Comment:
I am not sure if it's true with `DataDir` being set. Check out `TestRequestMessagesFailures`.

Reply:
Yup, that comment doesn't belong there.

Since the logic we want to test is in `activateShhService`, we do need to pass a dir. Ideally this file would make use of the `e2e.MakeTestNodeConfig` method, since it already contains logic to populate the `DataDir` also on Windows. Maybe we should move `e2e.MakeTestNodeConfig` to `github.com/status-im/status-go/t/utils`?

Reply:
That sounds good. Also, it may be beneficial to use https://golang.org/pkg/io/ioutil/#TempDir to clean up all artifacts after running the test.

Reply:
The artifacts in this directory (used by `MakeTestNodeConfig`) are normally kept from run to run, right? An example would be https://github.com/status-im/status-go/blob/feature/whisperv6-#665/t/destructive/peers_test.go#L34